### ISSUE TITLE
Not working anymore on Deluge 2.0.3

### ISSUE THREAD

**Issue by neoOpus** [YodaDaCoda/chrome-deluge-remote#24]
Hi

I have Deluge v 2.0.3 installed on my Synology NAS and since it was updated lately to this version (before I think it was just 1.x.x) this extension doesn't work anymore as used to be faithfully... nor the delugesiphon which point that something is broken on the association, I tried older chrome version to verify if it isn't due to my browser (Edgium) or more recent version of browsers but the issue remains... so, I guess something it has broken.

**Comment by Devo-McDuff** [comment]
Same issue, looks like project is dead and no alternatives. Hopefully someone will decide to fork at one point as it's a great little extension.

**Comment by doctorkb** [comment]
I have Deluge 2.0.3 and both this and delugesiphon work fine.  That said, Chrome doesn't like either of them anymore (I think it's part of their manifest version change issue that also breaks uBlock Origin) and there are hoops to jump through to fix that.

**Comment by markmand94** [comment]
Hi everyone,

For a while now, I’ve been frustrated that "Remote Deluge" no longer worked on Chrome. I’ve updated the code so that everyone can use it again.

I hope you find it useful! If you encounter any issues or have suggestions for improvements, please feel free to send me an email at:

markmand94@gmail.com
Download Link: https://share.markmand.org/download.php?file=chrome-Remote-Deluge_v1.2.41.zip

Best regards,

Mark Christiansen

**Comment by doctorkb** [comment]
> Hi everyone,
> 
> For a while now, I’ve been frustrated that "Remote Deluge" no longer worked on Chrome. I’ve updated the code so that everyone can use it again.
> 
> I hope you find it useful! If you encounter any issues or have suggestions for improvements, please feel free to send me an email at:
> 
> [markmand94@gmail.com](mailto:markmand94@gmail.com) Download Link: https://share.markmand.org/download.php?file=chrome-Remote-Deluge_v1.2.41.zip

Awesome! Works great! Any plans to release it through the Chrome Webstore?

**Comment by markmand94** [comment]
It's not free to upload a project to chrome webstore.
It's sad, but I can only put it on some free sites.
Hope you are enjoying it again.

**Comment by doctorkb** [comment]
I think it's only a $5 one-time developer fee.  I'm happy to contribute at least that much...

**Comment by markmand94** [comment]
Yeah, I can see its only 5$, if you want to pay, i will upload it on chrome webstore.
paypal: markmand94@gmail.com

**Comment by TSA3000** [comment]
Hey
Could you please give this one a try
https://chromewebstore.google.com/detail/deluge-remote-modern/okjmppgdfmooahhabhcdjdhnmnkgkila
